### DISCUSSION TITLE
Zwave JS with multiple controllers

### DISCUSSION THREAD

**Discussion by zebrahosting** [hassio-addons/addon-zwavejs2mqtt#501] [General]
My current house is running Homeseer with about 4 Z-net boxes. (Because of it's crazy length). I am planning to move over to HA but need to find a way to get it working with the networked Zwave.
SO first question is: can the Zwave JS addon run with tcp input? 
Second question, can there be multiple input points. Preferable reporting to the same mqtt / Zwave JS UI?
With Zigbee I have 2 addons running next to each other. May be something similar is possible with Zwave if needed?

**Comment by conradzik** [comment]
Commenting to say I am having a similar ask.

I would like to add two Z-Net controller's to a single Zwave JS UI installation. The distance between both locations is far too long to rely on repeaters to bridge the distance.
